submit_cgap/submission.py

```diff
@@ -108,39 +108,48 @@ def get_defaulted_institution(institution, user_record):
     """
     Returns the given institution or else if none is specified, it tries to infer an institution.
 
+    NOTE: As of the new permissions change, there is no way to infer an institution. One might be added later.
+          For now, there we just raise an error if no explicit institution was supplied.
+
     :param institution: the @id of an institution
     :param user_record: the user record for the authorized user
     :return: the @id of an institution to use
     """
-
+    ignored(user_record)
     if not institution:
-        submits_for = user_record.get('submits_for', [])
-        if len(submits_for) == 0:
-            raise SyntaxError("Your user profile declares no institution"
-                              " on behalf of which you are authorized to make submissions.")
-        elif len(submits_for) > 1:
-            raise SyntaxError("You must use --institution to specify which institution you are submitting for"
-                              " (probably one of: %s)." % ", ".join([x['@id'] for x in submits_for]))
-        else:
-            institution = submits_for[0]['@id']
-            show("Using institution:", institution)
+        # Ref: https://hms-dbmi.atlassian.net/browse/C4-371
+        raise SyntaxError("There not currently a way to infer an institution for user,"
```

Comment:
`There is not...` ? and I think there is now, it's `user_institution` - but Sarah can comment more specifically on this I think

Reply:
Yes, I created this field after I created the ticket and didn't remember to update the ticket. Ideally we would like to use user_institution, but can use --institution in the meantime if that change would take longer to make and merge.

Reply:
I think it won't take long to fix this.

Reply:
This should be set.